.ddev/config.yaml

```diff
@@ -0,0 +1,48 @@
+name: mass
+type: drupal9
+docroot: docroot
+php_version: "7.4"
+webserver_type: apache-fpm
+omit_containers:
+  - db
+  - dba
+disable_settings_management: true
+router_http_port: "80"
+router_https_port: "443"
+xdebug_enabled: false
+project_tld: local
+use_dns_when_possible: true
+web_environment:
+  - DOCKER_ENV=devel
+  - MYSQL_USER=circle
+  - MYSQL_PASSWORD=circle
+  - MYSQL_DATABASE=circle
+  - MYSQL_HOST=dbmass
+  - MEMCACHED_HOST=memcached
+  - DRUSH_OPTIONS_URI=https://mass.local:$DDEV_ROUTER_HTTPS_PORT
+  - DTT_BASE_URL=http://mass-web
+  - DTT_API_URL=http://chromedriver:9515
+#  - BROWSERTEST_OUTPUT_DIRECTORY=/tmp
+  - BROWSERTEST_OUTPUT_BASE_URL=https://mass.local:$DDEV_ROUTER_HTTPS_PORT
+  # Acquia Cloud - v2
+  - AC_API2_KEY
+  - AC_API2_SECRET
+  # New Relic. Needed for deployments.
+  - MASS_NEWRELIC_APPLICATION
+  - MASS_NEWRELIC_KEY
+  # Acquia basic auth.  Needed for tests on Acquia environments.
+  - LOWER_ENVIR_AUTH_USER
+  - LOWER_ENVIR_AUTH_PASS
+  # For Cloudflare Terraform.
+  - CLOUDFLARE_EMAIL
+  - CLOUDFLARE_TOKEN
+  # Specify a server for Xdebug in CLI (PhpStorm)
+#  - PHP_IDE_CONFIG=serverName=mass.local
```

Comment:
I added this because it's needed for running Xdebug in CLI with PhpStorm. It doesn't seem to work in the .env file but if this isn't a good spot, we might be able to add it in a docker-compose.env.yaml: https://mglaman.dev/blog/xdebug-over-command-line-ddev

Reply:
DDEV now adds this automatically in latest rc1 - https://github.com/drud/ddev/pull/3149

Reply:
Verified that running a drush command after sshing into the container works for Xdebug.